s3-backup/run.sh

```diff
@@ -1,26 +1,38 @@
 #!/usr/bin/with-contenv bashio
 # ==============================================================================
-# Home Assistant Community Add-on: Amazon S3 Backup
+# Home Assistant Community Add-on: S3 Backup
 # ==============================================================================
 #bashio::log.level "debug"
 
-bashio::log.info "Starting Amazon S3 Backup..."
-
 bucket_name="$(bashio::config 'bucket_name')"
 storage_class="$(bashio::config 'storage_class' 'STANDARD')"
-bucket_region="$(bashio::config 'bucket_region' 'eu-central-1')"
 delete_local_backups="$(bashio::config 'delete_local_backups' 'true')"
 local_backups_to_keep="$(bashio::config 'local_backups_to_keep' '3')"
 monitor_path="/backup"
 jq_filter=".backups|=sort_by(.date)|.backups|reverse|.[$local_backups_to_keep:]|.[].slug"
 
-export AWS_ACCESS_KEY_ID="$(bashio::config 'aws_access_key')"
-export AWS_SECRET_ACCESS_KEY="$(bashio::config 'aws_secret_access_key')"
-export AWS_REGION="$bucket_region"
+bashio::log.info "Configure S3 Backup..."
+aws configure set aws_access_key_id "$(bashio::config 'aws_access_key')"
+aws configure set aws_secret_access_key "$(bashio::config 'aws_secret_access_key')"
+
+aws configure set plugins.endpoint awscli_plugin_endpoint
+if bashio::config.has_value "endpoint_url"; then
+    endpoint_url="$(bashio::config 'endpoint_url' '')"
+    bashio::log.info "Configure S3 Backup endpoint to ${endpoint_url}"
+    aws configure set s3.endpoint_url "${endpoint_url}"
+    aws configure set s3api.endpoint_url "${endpoint_url}"
```

Comment:
I would highly prefer having this set as cli parameters. This way we can output pretty much all used parameters on line (https://github.com/thomasfr/hass-addons/pull/5/files#diff-06c5463fab0981ee606b95f4aee978538574d4d7a9884998c868a4f6370273a6R34)